### DISCUSSION TITLE
Why do I get "SyntaxError: Unexpected token < in JSON at position 0" when I try a simple fetch with useSwr?

### DISCUSSION THREAD

**Discussion by vonwao** [vercel/swr#652] [General]
It works fine when I use axios:

`const response = await Axios.get("/mock-api/mockItem");  //response.data has expected response
`

When I try the same trivial case with SWR:

`const { data, error } = useSWR("/mock-api/mockItem");
`
It gets stuck in the loading state.

I tried passing a simple fetch function (in case that is needed) like:

```
async function myFetch(...args) {
    // @ts-ignore
    const res = await fetch(...args);
    return res.json();
  }

```
and now I get "SyntaxError: Unexpected token < in JSON at position 0"

I'm probably making some dumb error, I know, but none of the examples are helping, they just confuse me more.

**Comment by sergiodxa** [comment]
Looks like your response is not a JSON but an HTML, check in your browser Network tab the actual response you are receiving from the server.
